README.md

```diff
@@ -26,13 +26,15 @@ If you are new to React Native, this is a helpful introduction: https://facebook
 
 4. Run `yarn nodeify`.
 
-5. Install required Pods by running `yarn install-pods`.
+5. Run `yarn install-bundle`.
 
-6. Run `yarn ios` to build the project for XCode.
+6. Install required Pods by running `yarn install-pods`.
 
-7. Open `rainbow-wallet/ios/RainbowWallet.xcworkspace`.
+7. Run `yarn ios` to build the project for XCode.
```

Comment:
Can we drop this? it seems to be not needed!

Also, I think it's worth mentioning that running bundler is important

Reply:
Will clear up the other inconsistencies after I meet with Bruno